universal-application-tool-0.0.1/conf/application.conf

```diff
@@ -105,7 +105,7 @@ play.modules {
 # to map through to internal messages, or allow the language to be stored in a cookie.
 play.i18n {
   # The application languages.
-  langs = [ "en-US", "zh-TW", "es-US", "vi", "so", "am", "ko", "tl" ]
```

Comment:
suggestion (blocking): Can we add a comment here to explain these should be alphabetized according to their English names?

Reply:
Thank you for the suggestion @carolinedanzi, I have added the comments and committed the changes here, please review and approve it.

Reply:
Thank you for making the change!